File: src/flutter/aula17_18/tipos_primitivos.dart
/*
  Tipos primitivos do Dart 
  - bool   : representa true ou false
  - int    : representa os números inteiros
  - double : representa os números de ponto flutuante
  - String : representa caracteres delimitados por ' ou "
  - List   : representa listas (similar ao array)
  - Set    : representa conjuntos
  - Map    : representa mapas (similar ao array associativo)
*/
void main() {
  print(10.runtimeType);
  print(1.0.runtimeType);
  print(true.runtimeType);
  print('um texto qualquer'.runtimeType);
  print([1, 2, 3].runtimeType);
  print({1, 2, 3}.runtimeType);
  print({1: '1', 2: '2', 3: '2'}.runtimeType);
}

File: src/kotlin/aula25_26/tipos_primitivos.kt
/*
  Tipos primitivos do Kotlin
  - Boolean                 : representa true ou false
  - Byte, Short, Int, Long  : representa os números inteiros
  - Float, Double           : representa os números de ponto flutuante
  - Char                    : representa caracteres delimitados por '
  - String                  : representa caracteres delimitados por "
  - Array                   : representa um array
*/
fun main() {
  println(10::class.simpleName)
  println(10L::class.simpleName)
  println(1.0f::class.simpleName)
  println(1.0::class.simpleName)
  println(true::class.simpleName)
  println('a'::class.simpleName)
  println("um texto qualquer"::class.simpleName)
  println(arrayOf(1, 2, 3)::class.simpleName);
}
